.changeset/lazy-geese-begin.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Add support for view-transition pseudo-elements
```

Comment:
```suggestion
Fixed: `selector-pseudo-element-no-unknown` false positives for `view-transition` pseudo-elements
```

To follow [our conventions](https://stylelint.io/maintainer-guide/pull-requests).